src/client/watchClientChanges.js

```diff
@@ -5,7 +5,7 @@ import webpackDevServer from 'webpack-dev-server';
 /**
  * Start webpack dev server for hmr
  */
-const watchClientChanges = clientConfig => {
+const watchClientChanges = (clientConfig, devServerOverrides, verbose) => {
```

Comment:
A couple of things here. Webpack already supports a [`devServer` option](https://webpack.js.org/configuration/dev-server/#devserver) through the standard webpack config. We should be able to use this instead of having to add our own overrides. 

Secondly re verbose logging, I think we can have a better way to control logging output than just placing a conditional around every console logs. I've done this [here](https://github.com/yusinto/pagerduty-webhook-processor/blob/master/src/logger.js) with my own logger class, but the idea is to be able to control log output from a central place. I think this should be a separate PR though, because it's unrelated to the your original intention of this PR which is to be able to customise dev server options.

Reply:
@yusinto Unless I misunderstand your comment, I don't think you understand what I'm trying to do here. The `devServerOverrides` is passed into your webpack dev server configuration that you have set up on the client. It allows a user of universal-hot-reload the ability to override the default configuration - you can see that here: https://github.com/yusinto/universal-hot-reload/pull/32/files#diff-799424c7a9cd2ab633aefd9fb1b04240R32-R44

For instance, if I want to turn on verbose stats instead of leaving them at the default of `errors-only` this `devServerOverrides` variable allows me to do that. Look at the whole PR in context and I think this should make a bit more sense.

Reply:
To give an example, I can now do something like:

```
UniversalHotReload({
  clientConfig,
  serverConfig,
  devServerOptions: {
    stats: 'verbose'
  }
});
```

If you look at the tests I wrote for this PR they show this capability and demonstrate that it's working as intended.

Reply:
@cereallarceny I understand clearly what you are trying to achieve. My suggestion was that consumer can set `clientConfig.devServer` and then we can use that object instead of having to specify a new param called `devServerOptions`. 

```tsx
UniversalHotReload({
  clientConfig, // this contains "devServer" object
  serverConfig,
  devServerOptions: {
    stats: 'verbose'
  }
});
```

Then in `watchClientChanges.js` we can merge this object with the hardcoded defaults `devServerOptions`. Does this make sense?